docs/REST API/rest-api.md

```diff
@@ -9138,6 +9141,125 @@ Add App Store (VPP) app purchased in Apple Business Manager.
 
 `Status: 200`
 
+### List Fleet library apps
+
+> **Experimental feature**. This feature is undergoing rapid improvement, which may result in breaking changes to the API or configuration surface. It is not recommended for use in automated workflows.
+
+List available Fleet library apps.
+
+`GET /api/v1/fleet/software/fleet_library_apps`
+
+#### Parameters
+
+| Name | Type | In | Description |
+| ---- | ---- | -- | ----------- |
+| team_id       | integer | query | **Required**. The team ID. Filters Fleet library apps to only include apps available for the specified team.  |
+
+#### Example
+
+`GET /api/v1/fleet/software/fleet_library_apps?team_id=3`
+
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "fleet_library_apps": [
+    {
+      "id": "1",
+      "name": "1Password",
+      "version": "8.10.40",
+      "platform": "darwin"
+    },
+    {
+      "id": "2",
+      "name": "Adobe Acrobat Reader",
+      "version": "24.002.21005",
+      "platform": "darwin"
+    },
+    {
+      "id": "3",
+      "name": "Box Drive",
+      "version": "2.39.179",
+      "platform": "darwin"
+    },
+  ]
+}
+```
+
+### Get Fleet library app
+
+> **Experimental feature**. This feature is undergoing rapid improvement, which may result in breaking changes to the API or configuration surface. It is not recommended for use in automated workflows.
+
+Returns information about the specified Fleet library app.
+
+`GET /api/v1/fleet/software/fleet_library_apps/:id`
+
+#### Parameters
+
+| Name | Type | In | Description |
+| ---- | ---- | -- | ----------- |
+| id   | integer | path | **Required.** The Fleet library app's ID. |
+| team_id             | integer | query | _Available in Fleet Premium_. Filters response data to the specified team. Use `0` to filter by hosts assigned to "No team" (default: 0).  |
```

Comment:
I see these parameters match the [Get Software](https://fleetdm.com/docs/rest-api/rest-api#get-software) endpoint, which is good. But I'm a bit confused by the `team_id` parameter. If we're requesting a software title by ID, there should be only one result, so there is no need to filter further with a `team_id`, unless we're re-using software IDs across teams? @georgekarrv Or maybe I'm missing another reason to have this.

Reply:
I forgot to remove this. I think we don't need it. It just returns app metadata, which is same for each team.